### PR TITLE
Don't fail on upload_results_to_bq not being defined

### PR DESCRIPTION
**Pull request by yashykt**
Don't fail on upload_results_to_bq not being defined

### CODE REVIEW

File: tools/run_tests/run_tests.py [hunk 1]
```diff
@@ -1931,8 +1931,11 @@ def _build_and_run(check_cancelled,
                     0],  # args.language is a list but will always have one element when uploading to BQ is enabled.
                 'platform': platform_string()
             }
-            upload_results_to_bq(resultset, args.bq_result_table,
+            try:
+              upload_results_to_bq(resultset, args.bq_result_table,
                                  upload_extra_fields)
+            except NameError:
+              pass # It's fine to ignore since this is not critical
```
Comment (mhaidrygoog) [inline tools/run_tests/run_tests.py]: Do we still want to log a warning or error message here?

Comment (gnossen) [inline tools/run_tests/run_tests.py]: Might want to do a `logging.warning(e)` just so we're aware that there's a problem.

Comment (yashykt) [reply tools/run_tests/run_tests.py]: Done!

Comment (yashykt) [reply tools/run_tests/run_tests.py]: Thanks for making the change easy :)

**Pull request merged**